query10.sql

```diff
@@ -4,4 +4,8 @@
 */
 
 -- Enter your SQL query here
-select ...
+select st_distance(
+  ST_Transform(st_setsrid(st_point(-75.19264,39.95222),4326), 3857),
+  ST_Transform(the_geom, 3857)
```

Comment:
As you know at this point, SRID 3857 will give a slightly distorted measure of distance at latitudes far from the equator.